app/services/billing-accounts/setup/submit-select-account.service.js

```diff
@@ -47,6 +47,12 @@ async function _save(session, payload) {
   return session.$update()
 }
 
+async function _submissionData(session, payload) {
```

Comment:
If you remove the async from this function it should sort out the SonarCloud issue "Expected a non-Promise value to be spread in an object."

```suggestion
function _submissionData(session, payload) {
```